scripts/initdb_and_run.sh

```diff
@@ -4,6 +4,7 @@ if [ ! -e /app/.cache/mysql/db.done ]; then
     echo "Initializing DB..."
     PYTHONPATH=${PYTHONPATH}:/app/vendor/lib/python python /app/migrate_repo/manage.py version_control mysql://shipit:shipitpw@shipitdb/shipit migrate_repo
     PYTHONPATH=${PYTHONPATH}:/app/vendor/lib/python python /app/migrate_repo/manage.py upgrade mysql://shipit:shipitpw@shipitdb/shipit migrate_repo
+    bunzip2 -c /app/scripts/sample-data.sql.bz2 | mysql -h shipitdb -u shipit --password=shipitpw shipit
```

Comment:
Funny you should suggest that method, it's how I updated the earlier export to the current schema. The one downside is there's no incentive to keep the data up to date.